src/components/screens/map-screen/map-screen.tsx

```diff
@@ -135,6 +137,12 @@ class MapScreen extends React.Component<Props, State> {
 
     this.getUserSavedRoutes();
     this.getUserFavouriteLocations().then(() => this.setLocationOptions());
+    if (this.mapRef.current != null) {
```

Comment:
```suggestion

    if (this.mapRef.current != null) {
```
Empty lines before if statements is usually a good idea